CONTRIBUTING.md file for Newcomers to easily start contributing. #466

end-9214 opened the pull request:
This will be very useful for newcomers.

Currently I have just added how to setup manager locally and start contributing  :D.

Kindly Check if theres any mistakes.

end-9214 commented:
Hey @rgaudin , I was trying to setup the scheduler but i m unable to. I tried using the same code in the README.md but got an error and then tried to run it in docker using `dockerfile` in a common network but it isn't pingable . Am i setting it up correctly or wrong?

rgaudin reviewed (changes requested):
Thank you for your PR.

Please rework this:

- Docker is used for production packaging ; not devel/contrib
- There's no info about the mandatory database. In your example you dont mount any volume so changes wont persist 
- There's a recipe for creating the test user already (see entrypoint)
- there are inconsistencuies in your doc: the username in the code sample and the instructions to log in are different
- there is no `views.py`
- this is just about the manager part
- there's no explanation about what the manager is about

end-9214 commented:
> Thank you for your PR.
> 
> Please rework this:
> 
>     * Docker is used for production packaging ; not devel/contrib
> 
>     * There's no info about the mandatory database. In your example you dont mount any volume so changes wont persist
> 
>     * There's a recipe for creating the test user already (see entrypoint)
> 
>     * there are inconsistencuies in your doc: the username in the code sample and the instructions to log in are different
> 
>     * there is no `views.py`
> 
>     * this is just about the manager part
> 
>     * there's no explanation about what the manager is about

fixed the following -
* mounts data volume for persistence
* using the automatically created user - admin
* removed inconsistencies 

>> I created this docker-compose.yml file to build and run the manager, scheduler and mongoDB database on the same network easily.
